carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/decorator/ExtendedWebElement.java

```diff
@@ -299,6 +300,43 @@ public WebElement getElement() {
         
         return this.element;
     }
+
+    private Optional<WebElement> findWebElement() {
```

Comment:
1. let's remove findWebElement and reuse existing findElement.
2. inside findElement we again try to remove global driver context and use only searchContext!
3.

Reply:
resolved